cmd/pipeline/deploy.go

```diff
@@ -175,14 +176,15 @@ func waitUntilRunning(ctx context.Context, name, namespace string, timeout time.
 }
 
 func getCurrentNamespace(ctx context.Context) (string, error) {
-	c, _, namespace, err := k8Client.GetLocal("")
+	c, _, err := k8Client.GetLocal("")
```

Comment:
it seems like half the calls  pass this as an empty string. Could we instead have a `k8Client.GetLocal()` and `k8Client.GetLocalWithContext(context string)`  to make it easier to read?